packagehandlers/nugetpackagehandler.go

```diff
@@ -0,0 +1,35 @@
+package packagehandlers
+
+import (
+	"github.com/jfrog/frogbot/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+const dotnetPackageUpgradeExtraArg = "package"
+
+type NugetPackageHandler struct {
+	CommonPackageHandler
+}
+
+func (dotnet *NugetPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityDetails) error {
+	if vulnDetails.IsDirectDependency {
+		return dotnet.updateDirectDependency(vulnDetails)
+	} else {
+		return &utils.ErrUnsupportedFix{
+			PackageName:  vulnDetails.ImpactedDependencyName,
+			FixedVersion: vulnDetails.SuggestedFixedVersion,
+			ErrorType:    utils.IndirectDependencyFixNotSupported,
+		}
+	}
+}
+
+func (dotnet *NugetPackageHandler) updateDirectDependency(vulnDetails *utils.VulnerabilityDetails) (err error) {
+	impactedPackage := strings.ToLower(vulnDetails.ImpactedDependencyName)
```

Comment:
I think that nuget packages are case sensitive. For example, I'm not sure that the command `dotnet add microsoft.entityframeworkcore -v 6.0.4` would work here. (The package name is `Microsoft.EntityFrameworkCore`)

Reply:
I have double checked it and it works. Also read about it online and saw it is case-insensitive